src/vs/editor/contrib/message/browser/messageController.css

```diff
@@ -37,6 +37,15 @@
 	border: 1px solid var(--vscode-inputValidation-infoBorder);
 }
 
+.monaco-editor .monaco-editor-overlaymessage .message p {
+	margin-block: 0px;
+}
+
+.monaco-editor .monaco-editor-overlaymessage .message a {
+	font-weight: bold;
```

Comment:
We should have a follow up UX discussion how discoverable `bold` is for links. I would have probably expected an underline at least.

Reply:
👍 I will initiate this.